static-site/content/resource-listing.yaml

```diff
@@ -58,12 +58,39 @@ coreGroupDisplayNames:
 # showcase tiles which will be shown is decided by the UI and changes with
 # screen size.
 coreShowcase:
-  - {name: "SARS-CoV-2",        img: "ncov.png",              filters: ["ncov"]}
-  - {name: "Seasonal Flu",      img: "seasonalinfluenza.png", filters: ["seasonal-flu"]}
-  - {name: "Avian Flu",         img: "avianinfluenza.png",    filters: ["avian-flu"]}
-  - {name: "mpox",              img: "mpox.png",              filters: ["mpox"]}
-  - {name: "RSV",               img: "rsv1.png",              filters: ["rsv"]}
-  - {name: "dengue",            img: "dengue.png",            filters: ["dengue"]}
-  - {name: "Ebola (West Africa)", img: "ebola.png",           filters: ["ebola"]}
-  - {name: "Measles",           img: "measles.jpg",           filters: ["measles"]}
-  - {name: "Zika",              img: "zika.png",              filters: ["zika"]}
+  - name: SARS-CoV-2
+    img: ncov.png
+    filters:
+      - ncov
+  - name: Seasonal influenza
+    img: seasonalinfluenza.png
+    filters:
+      - seasonal-flu
+  - name: Avian Flu
+    img: avianinfluenza.png
+    filters:
+      - avian-flu
+  - name: Mpox
+    img: mpox.png
+    filters:
+      - mpox
+  - name: RSV
+    img: rsv1.png
+    filters:
+      - rsv
+  - name: Dengue
+    img: dengue.png
+    filters:
+      - dengue
+  - name: Ebola (West Africa)
```

Comment:
Switch to `Ebola in West Africa`

I don't like the unnecessary parentheticals.

Reply:
Fixed in [force-push](https://github.com/nextstrain/nextstrain.org/compare/36af2e91337f13b1a1da138d2dfef5d040b37288..833b1146cfdbb9af1e8eb207a1eb964b76967ae4)